flaskr.py

```diff
@@ -48,23 +48,29 @@ def close_db(error):
 @app.route('/')
 def show_entries():
     db=get_db()
-    cur = db.execute('select id, title, text from entries order by id desc')
+    cur = db.execute('select id, title, text from entries order by sort_order desc')
     entries = cur.fetchall()
     return render_template('show_entries.html', entries=entries)
 
+
 @app.route('/add', methods=['POST'])
 def add_entry():
     if not session.get('logged_in'):
         abort(401)
     db = get_db()
+    max_order_id = db.execute('SELECT MAX(sort_order) FROM entries').fetchone()[0]
+    if max_order_id is None:
+        new_order_id = 10
```

Comment:
Why `10`? Why not `78` or something else? I would expect `1` which is not a magic number (https://en.wikipedia.org/wiki/Magic_number_(programming)#Unnamed_numerical_constants).